### DISCUSSION TITLE
New recordings with no Audio stream

### DISCUSSION THREAD

**Discussion by techiegirl2day** [OpenSageTV/Community#31] [Windows]
I have 2 new show files that have only video & no audio ( checked metadata & codecs ) via VLC.  My guess is these were due to Bell using a newer codec that SageTV cant access?  I checked audio. Also checked in my Firestick client. Only common denominator is the lack of audio track. Everything works fine on live channels & other recordings from today are fine, except for shows from CTV/Bell.  Is this newer audio encrypted?  What could be happening & are we SOL?

I found this old post to attempt troubleshooting my configs
[(https://forums.sagetv.com/forums/showthread.php?t=67048&highlight=audio+formats)](url)

Galaxysurfer

**Comment by LoisGNS** [comment]
I've occasionally had that happen, but the cause might be completely different. I think in each case I traced it to my Comcast DTA, which had gotten itself in some mode or other (accidentally set to mute, for example)  and was reset by cycling the power. Were the good recordings made on the same tuner plus cablebox/card (if applicable) as the bad ones? I wasn't sure if your Firestick client was trying to play directly from CTV/Bell and not getting audio, or if it was playing a Sage recording or live via sage miniclient.

**Comment by wayner9** [comment]
What is the capture device being used?  An HD-PVR, OTA, etc?

**Reply by techiegirl2day** [reply]
I am in Canada so no cable card involved. I am using a SiliconDust Flex 4K which is  a OTA tuner. The only other variables I can think of is antenna signal issues and /or a recent firmware update. It just seems odd that only that one channel is affected. It also screwed with recording the new Marvel Movie starring Simu Lui. So don't know if it is signal issues or something else. Will tinker with signal to improve strength and go from there. Unfortunately stuck with firmware upgrade since it isn't reversible.

**Reply by wayner9** [reply]
I am in Canada as well (Toronto) and I don't have that device, but I do have two other older Silicon Dust OTA devices.  I will try to record a show on CTV and see if I have any issues.  Can you determine if there is any audio at all - like using a program that tells you what codecs and bitrates are being used.

**Reply by techiegirl2day** [reply]
It was only a specific show.  Was wondering if they are using some sort of flag on shows? It was a bit bizarre.
